gnosis/cowswap
# [Profile] Add explorer orders link to Profile page and app menu

# Summary

Fixes #1883 

Added a link to Orders explorer from Profile page and App menu

<img width="1511" alt="Screen Shot 2021-11-28 at 21 36 49" src="https://user-images.githubusercontent.com/11525018/143793097-5b699c49-d115-4500-bb7d-81afe3aa32bf.png">

  # To Test

1. Open profile page
- [x] Connect an account
- [x] You'll see a View all orders link 
2. Open app menu clicking on the three dots in the upper right corner
- [x] Connect an account
- [x] You'll see a View all orders link
